src/common/expression-parser/validateAggregate.ts

```diff
@@ -0,0 +1,108 @@
+import { ANY_TYPES, NUMERICS } from "$lib/duckdb-data-types";
+import type { ExprCall } from "pgsql-ast-parser";
+import type { ProfileColumn } from "$lib/types";
+
+// Taken from https://duckdb.org/docs/sql/aggregates
+export const AllowedAggregates: {
+  [agg in string]: Array<Set<string>>;
+} = {
+  // number, any
+  arg_max: [NUMERICS, ANY_TYPES],
+  argMax: [NUMERICS, ANY_TYPES],
+  max_by: [NUMERICS, ANY_TYPES],
+  arg_min: [NUMERICS, ANY_TYPES],
+  argMin: [NUMERICS, ANY_TYPES],
+  min_by: [NUMERICS, ANY_TYPES],
+
+  // number
+  count: [NUMERICS],
```

Comment:
i think we should be able to `count` and `count distinct` any type, right?

Reply:
approx_count_distinct too?

Reply:
Good point. Fixed this used case.